gp2gp-translator/src/main/java/uk/nhs/adaptors/pss/translator/mapper/AllergyIntoleranceMapper.java

```diff
@@ -88,14 +92,18 @@ private AllergyIntolerance mapAllergyIntolerance(RCMRMT030101UKEhrComposition eh
                 .getId()
                 .getRoot();
 
+        final Meta meta = CONFIDENTIALITY_CODE_PRESENT.test(observationStatement) ?
```

Comment:
I _suspect_ this refactoring won't be the most desirable once we've implemented a few of these mappers, but given my mindset of refactoring when the duplication is indeed visible and a problem, let's leave it as it is for now.